src/Kliva/Controls/ActivityFeedControl.xaml.cs

```diff
@@ -68,5 +69,35 @@ private void ScrollToTop()
         {
             _scrollViewer?.ScrollToVerticalOffsetWithAnimation(0);
         }
+
+        private void ActivityList_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            if (IsDesktopState())
+            {
+                // Only update the ViewModel if we are in the wide state where the list supports
+                // selection
+                ViewModel.SelectedActivity = (ActivitySummary)e.AddedItems.FirstOrDefault();
```

Comment:
Doing this manually instead of using two-way binding improves scenarios where you go from mobile to desktop view by allowing you to maintain context of the selected item. Otherwise the selected item will always get clobbered by the two-way binding when you go from mobile <-> desktop.